test/parallel/test-fs-read-stream-inherit.js

```diff
@@ -48,61 +49,61 @@ let paused = false;
 }
 
 {
-  const file3 = fs.createReadStream(fn, Object.create({encoding: 'utf8'}));
-  file3.length = 0;
-  file3.on('data', function(data) {
+  const file = fs.createReadStream(fn, Object.create({encoding: 'utf8'}));
+  file.length = 0;
+  file.on('data', function(data) {
```

Comment:
By lambdas you mean `() => {}` (fat arrow) functions? I don't think we have an official policy (although I personally prefer fat arrow functions because they seem simpler).